content/articles/building-an-ensemble-learning-based-regression-model-using-python/index.md

```diff
@@ -0,0 +1,173 @@
+﻿
+
+
+### Building an Ensemble Learning Based Regression Model Using Python
+
+Machine learning models are always evaluated based on their performance using specific metrics like; accuracy, precision, Mean Squared Error(MSE) etc. Each type of machine learning problem has its own evaluation metrics. 
+Building high performance models (models with low errors) therefore, depends on how good the evaluation metric score is.
+
+### Introduction 
+
+In this tutorial, we shall be building a performance driven linear regression model using ensemble learning.
+
+ **Linear regression** is a statistical method of modeling the relationship between independent variables (x) and dependent variables (y). It uses independent variables (features) to predict dependent variables (target).
+
+**Ensemble learning** is a machine learning technique that seeks to achieve better predictive performance of model by combining decisions from different models.
+
+For our model's evaluation, we shall be using RMSE (Root Mean Squared Error). 
+
+> **NB:** Regression problems cannot be measured using accuracy metric since the goal is to measure how close the predicted values is to the expected values and not to evaluate how correct the prediction is. Hence we use errors to evaluate our models.
+
+### Prerequisites
+
+To follow through the tutorial, you need to:
+1. Know the basics of Python
+2. Have a [Kaggle](https://www.kaggle.com/) account
+3. Know basics of Machine Learning
+
+### Setting Up Your Environment
+
+Before we start building our model, we shall first go to Kaggle and create a new **notebook** and rename it to **Create_Folds**.
+![notebook](engineering-education/building-an-ensemble-learning-based-regression-model-using-python/notebook.jpg)
+ 
+After that, [download](https://www.kaggle.com/c/30-days-of-ml/data) the data from Kaggle and add it to your environment using the **Add Data** button and upload the downloaded data as **Dataset**.
```

Comment:
Please include screenshots for this section too. It would allow the reader to follow through easily